imports/plugins/included/ui-search/lib/containers/searchSubscription.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import _ from "lodash";
+import { Meteor } from "meteor/meteor";
+import * as Collections from "/lib/collections";
+import { Components, composeWithTracker } from "@reactioncommerce/reaction-components";
+import SearchModal from "../components/searchModal";
+
+class SearchSubscription extends Component {
+  render() {
+    return (
+      <SearchModal {...this.props}/>
+    );
+  }
+}
+
+function getSiteName() {
+  const shop = Collections.Shops.findOne();
+  return typeof shop === "object" && shop.name ? shop.name : "";
+}
+
+function getProductHashtags(productResults) {
+  const hashtags = [];
+  for (const product of productResults) {
```

Comment:
this nested loop would be much better as an `Array.prototype.reduce`

Reply:
True, using a `reduce` would have been better.